components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -22,7 +22,7 @@
 from omero.rtypes import wrap
 from omero.testlib import ITest
 from omero.gateway import BlitzGateway, KNOWN_WRAPPERS
-from omero.model import ScreenI, PlateI, WellI, WellSampleI
+from omero.model import ScreenI, PlateI, WellI, WellSampleI, DatasetI, ImageI
```

Comment:
Just to save yourself some pain later on, you might want to go ahead and have the model object imports one per line in alphabetical order. It'll save on merge conflicts and make it easier to review going forward. I ended up doing it late for omero-marshal, where we have 20+ model objects being imported in `conftest.py`, and I really wished I had done it sooner.